internal/command/jsonstate/state.go

```diff
@@ -518,16 +538,19 @@ func SensitiveAsBool(val cty.Value) cty.Value {
 		vals := make([]cty.Value, 0, length)
 		it := val.ElementIterator()
 		for it.Next() {
-			_, v := it.Element()
-			vals = append(vals, SensitiveAsBool(v))
+			kv, ev := it.Element()
+			path := append(path, cty.IndexStep{
+				Key: kv,
+			})
```

Comment:
Does `IndexStep` work fine with `Tuple`? From the `IndexStep` doc:

> IndexStep is a Step implementation representing applying the index operation to a value, which must be of either a list, map, or set type.

Reply:
Yes, see https://github.com/zclconf/go-cty/blob/15a9d85f4808a6d2a34baf813f3806c8e3d334c7/cty/path.go#L216C1-L219C4
and I also add a test for tuple